src/main.ts

```diff
@@ -26,10 +26,10 @@ const createRecipesInput = () => {
 }
 
 const showIngredients = () => {
-  const receivedIngredients: string[] = Storage.get('ingredients')
+  const receivedIngredients = Storage.get('ingredients')
```

Comment:
im still looking for generic functon with TypeScript ;)

Reply:
i don't think that get() could work as generic function. It's always get a string data what was json.stringified and always needs to json.parses it

Reply:
ok, app works but typescript does not allow me to compile program